### PR TITLE
Update address and loanRequest function.

### PR DESCRIPTION
**Pull request by ashish-hashstack**
Signed-off-by: Ashish Kumar Mishra <ashish10677@gmail.com>

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -1,5 +1,5 @@
 REACT_APP_API_URL="https://testapi.hashstack.finance/"
 
-REACT_APP_DIAMOND_ADDRESS = "0x6004fd80569f8EDB7e10F2Df80Aa690819Ba2Dbf"
+REACT_APP_DIAMOND_ADDRESS = "0x44D65039Ef9bDA3c4BdC42C74E9CAA4C81Da143b"
```
Comment (MPankajArun) [inline .env.development:3]: Update this in .env.production

Comment (ashish10677) [reply .env.development:3]: It's done

Comment (ashish-hashstack) [reply .env.development:3]: Done

File: src/blockchain/web3/Loan.ts [hunk 2]
```diff
@@ -31,7 +31,20 @@ class LoanWrapper {
     }
 
     loanRequest(market: string, commitment: string, loanAmount: number, loanDecimal: number, collateralMarket: string, collateralAmount: number, collateralDecimal: number) {
-        return this.loan1.send("loanRequest", {}, market, commitment, NumToBN(loanAmount, loanDecimal), collateralMarket, NumToBN(collateralAmount, collateralDecimal));
+        let marketAddress = pancakeSwapTokenAddress[market];
+        let API_URL = `https://testapi.hashstack.finance/seedTokenPrice?marketAddress=${marketAddress}&market=${market}&amount=${loanAmount}&decimal=${loanDecimal}`
```
Comment (MPankajArun) [inline src/blockchain/web3/Loan.ts:35]: create this URL with help process.env.REACT_APP_API_URL

**Pull request merged**